client/web/compose/src/components/Admin/Module/UniqueValues.vue

```diff
@@ -0,0 +1,220 @@
+<template>
+  <div>
+    <div
+      v-for="(rule, index) in rules"
+      :key="index"
+    >
+      <hr v-if="index">
+      <div>
+        <div class="d-flex justify-content-between align-items-center">
+          <h5>{{ $t("uniqueValueConstraint") }} #{{ index + 1 }}</h5>
+
+          <div class="px-4">
+            <c-input-confirm
+              @confirmed="rules.splice(index, 1)"
+            />
+          </div>
+        </div>
+        <b-form-checkbox
+          v-model="rule.strict"
+          switch
+          class="mt-3"
+        >
+          {{ $t("allowRecordsSave") }}
+        </b-form-checkbox>
+      </div>
+      <div class="mt-3">
+        <!-- <b-form-group>
+          <b-form-input :placeholder="$t('warningMessage')" />
+        </b-form-group> -->
+        <div
+          v-if="rule.constraints.length > 0"
+          class="table-responsive"
```

Comment:
By replacing table with b-table-simple you dont need the class you can instead use the props. Same for the line below